packages/back-end/src/services/athena.ts

```diff
@@ -76,6 +76,17 @@ export async function runAthenaQuery(
   const retryWaitTime =
     (parseInt(process.env.ATHENA_RETRY_WAIT_TIME || "60") || 60) * 1000;
 
+  const resultReuseSettings =
+    typeof conn.resultReuseMaxAgeInMinutes === "undefined" ||
+    conn.resultReuseMaxAgeInMinutes === "0"
+      ? {
+          Enabled: false,
+        }
+      : {
+          Enabled: true,
+          MaxAgeInMinutes: parseInt(conn.resultReuseMaxAgeInMinutes),
+        };
```

Comment:
This can be made a little less risky with some more typeguards.  Like the value might be an empty string or `parseInt` might fail.

Also, it's less risky if instead of always passing in `ResultReuseConfiguration`, we only pass it in when it's enabled.  That way, the baseline code remains completely unchanged for existing users and there's no risk of it breaking unintentionally.

Reply:
@jdorn @Kevin-Chant Do you have any updates on the progress of this feature?

Reply:
Hi @fabianmarinog, right now the team is focused on finishing features for our 3.4 release, but after the code freeze (~25th) I planned to get a testing environment set up so I could verify these changes and get it merged. So no updates yet but hopefully it should be relatively quick once I can test that it works the way I expect it to :+1: